Project EFT/Controllers/ProblemController.cs

```diff
@@ -74,6 +74,33 @@ public IActionResult CheckAnswer()
             return View("Problem");
         }
 
+        public IActionResult AddProblemPage()
+        {
+            
+            return View();
+        }
+
+
+        [HttpPost]
+        public IActionResult AddProblem()
+        {
+            Problem problem = new Problem(0, Request.Form["Title"], Request.Form["question"], Request.Form["answer"], 0, 0);
+
+            if(problem.Title.Equals("") || problem.Question.Equals("") || problem.Answer.Equals(""))
+            {
+                ViewData["message"] = "One of the entries was left blank, please try again.";
+            }
+            else
+            {
+                DBConnector.InsertNewProblem(problem);
```

Comment:
what happens if `InsertNewProblem` fails?

Reply:
In reference to the next line, which got cut off in the MLC:
`ViewData["message"] = "Your new problem has been added to the list of problems!";`

Reply:
Now it checks what is returned, true/false, and sends a message accordingly.